documentation/navitia.io/source/integration.rst

```diff
@@ -1978,7 +1978,9 @@ date_time
 +==========================+======================+================================+
 | additionnal_informations | Array of String      | Other information: TODO enum   |
 +--------------------------+----------------------+--------------------------------+
-| date_times               | Array of String      | Date time                      |
+| departure_date_times     | Array of String      | Date time                      |
```

Comment:
isn't it `departure_date_time` with only one date time ?
Actually I don't understand this, didn't you want to change the stop_date_time section to split `date_time 	date-time 	A date time` into a departure/arrival field ?